addons/hr_expense/tests/test_expenses.py

```diff
@@ -194,7 +194,7 @@ def test_expense_values(self):
             {
                 'debit': 652.17,
                 'credit': 0.0,
-                'amount_currency': 1304.348, # untaxed amount
+                'amount_currency': 1304.348,  # untaxed amount
```

Comment:
Even if you right: it should be done that way, you should avoid such modification.
Reasons are:
- The dumb one: people will tell you it's a useless diff:
if they browse the file commit history, it's just noise and it requires them to browse the file previous to your commit to know why the line was there in the first place.

- The good one: forward-port conflicts:
If someone modify this line in 15.0 and its modification gets forward-ported, the change you are making would generates a conflict that would require a manual intervention from the guy fixing in 15.0.

Generally speaking, we try to avoid style modification.
Do it only if it consequently improves readability.

Reply:
VS Code is automatically formatting the files according to PEP8, that enforces at least 2 spaces before an inline comment.
That's why it was automatically changed. I now configured my formatter to not automatically format on save and reverted these changes.